index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <h1>Hello Microverse!</h1>
```

Comment:
- [Optional] As the previous reviewer noted, The  `head` element is a container for metadata (data about data).

- The tag defines the document's body. The element contains all the contents of an HTML document, such as headings, paragraphs, images, hyperlinks, tables, lists, etc.

- The `title` tag should be used here instead, and the `h1` in the body of your project